polyfill/runtest262.mjs

```diff
@@ -4,7 +4,7 @@ const result = runTest262({
   test262Dir: 'test262',
   polyfillCodeFile: 'script.js',
   expectedFailureFiles: ['test/expected-failures.txt'],
-  timeoutMsecs: process.env.TIMEOUT || 30000,
+  timeoutMsecs: process.env.TIMEOUT || 60000,
```

Comment:
I don't think it's this PR that's making timezone-heavy tests run slower, but they are now bumping up against the lower timeout so extending it here.